tests/factories/file.js

```diff
@@ -3,25 +3,33 @@ import faker from 'faker';
 
 FactoryGuy.define('file', {
     default: {
-        name: () => faker.lorem.word() + '.txt',
+        name: () => faker.lorem.words(1) + '.txt',
```

Comment:
I was getting errors with `lorem.word`, so I guess I have an old version? I have `3.1.0` installed, but [it looks like](https://github.com/johnotander/ember-faker/blob/master/bower.json#L16) `ember-faker` depends on `~2.1.0`... could that be messing with things? I am consistently bewildered by versioning issues.